### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_Reflow: Data is getting lost at 400% zoom

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4828]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-SH_AzureDataStudioSQL_Feb2021;-SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;-WCAG3.3.2;-JAWS;-NVDA;-STP;-AINotInScope;-Zoom;

--- Prerequisite: 

- Set resolution to 1280*1024
- Set scaling to 400

--- Environment Details:
Application Name: Azure Data Studio
Version: 1.25.3-insider
Windows Version: Windows10

--- Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Check if data is getting lost at 400% zoom

--- Actual:
Data is getting lost at 400% zoom. The radio buttons "SQL Database" and "SQL Edge" are not properly visible also the edit fields are not properly visible.

Same issue is also observed throughout the application

--- Expected:
All the data should be visible at 400% zoom

--- User Impact:
Low Vision users will face difficulty in getting the data.
